regular-expressions/introduction/basic-syntax-ii/regex-anchors.md

```diff
@@ -21,29 +21,33 @@ links:
 ---
 ## Content
 
-So far we've written regular expressions that can *partially match*[1] across large pieces of text, but this isn't always desirable!
+So far we've written regular expressions that can *partially match*[1] pieces of text, but this isn't always desirable!
 
 Imagine if we wanted to match the word `success`, but our text also included the word `unsuccessful`. Our pattern would match on this too...
 
 This is where **anchors** come in handy.
 
-> 💡 The `^` is used to signify the *start of a string*, and the `$` character is used to signify the *end of a string*.
+> 💡 The '^' is used to signify the *start of a string*, and the '$' character is used to signify the *end of a string*.
 
-`/^success/`
+```
+/^success/
+```
 
-The string must start with `'success'`.
+The string must start with `success`.
 
-'**success**' ✅
-'unsuccessful' ❌
+`success` ✅
+unsuccessful ❌
 
-`/.+\.$/`
+```
+/.+\.$/
+```
 
-The string can have one or more character but must end with a `'.'`.
+The string can have one or more character but must end with a '.'.
```

Comment:
```suggestion
The string can have one or more character but must end with a `'.'`.
```

looks a bit weird in the app without the back ticks

![image](https://user-images.githubusercontent.com/19802557/83124237-d88bc580-a0cd-11ea-8862-72ac60d0cfb2.png)